### DISCUSSION TITLE
Hosting FireFly III on Railway

### DISCUSSION THREAD

**Discussion by Jo9871** [firefly-iii/firefly-iii#7813] [Firefly III]
I have been trying for the past few hours to host FireFly III on [Railway](https://railway.app).
I have tried using the both the `docker-compose.yml` and `Dockerfile` files, with no luck.

I was wondering if anyone has been able to get FireFly III to work on Railway or not. If so any help would be appreciated.

**Comment by JC5** [comment]
I've never used Railway before. But is there anything specific you're running into?
